pkgs/by-name/ko/komac/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, SystemConfiguration
+, testers
+, komac
+}:
+
+let
+  pname = "Komac";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

`The pname and the version attribute must not contain uppercase letters — e.g., "mplayer" instead of "MPlayer".`

and maybe `pname` doesn't need to be in `let`, although it's just personal taste

Reply:
I didn't know the no uppercase rule, thank you for the pointing out!